typescript/cli/src/commands/registry.ts

```diff
@@ -88,3 +92,88 @@ const addressesCommand: CommandModuleWithContext<{ name: string }> = {
     }
   },
 };
+
+const chainTargetsCommandOption: Options = {
+  type: 'string',
+  description: 'Comma-separated list of chain names',
+  alias: 'c',
+  // TODO choices: ... can we configure this so that it's a list of chain names including any new chains added to the local registry
+};
+
+const environmentCommandOption: Options = {
+  type: 'string',
+  description: 'The name of the environment to deploy to',
+  alias: 'e',
+  choices: ChainTypes,
+};
```

Comment:
Recommend moving these to commands/options and de-duping with similar coreTargetsCommandOption